db/migrate/20251110162648_add_api_updated_at_to_training_periods.rb

```diff
@@ -0,0 +1,8 @@
+class AddAPIUpdatedAtToTrainingPeriods < ActiveRecord::Migration[8.0]
+  disable_ddl_transaction!
+
+  def change
+    add_column :training_periods, :api_updated_at, :datetime, null: false, default: -> { "CURRENT_TIMESTAMP" }
```

Comment:
Would `api_transfer_updated_at` make more sense here? usually we reserve `api_updated_at` for the endpoint that returns the model, but we don't have one for `TrainingPeriod` so it may be a bit confusing as to where its used?